src/stan/variational/advi_params_fullrank.hpp

```diff
@@ -0,0 +1,112 @@
+#ifndef STAN_VARIATIONAL_ADVI_PARAMS_FULLRANK__HPP
+#define STAN_VARIATIONAL_ADVI_PARAMS_FULLRANK__HPP
+
+#include <stan/math/prim/mat/fun/Eigen.hpp>
```

Comment:
oh i see.

i think i also need to make some updates to the variational tests. i'll push those soon too.